assemblyline/common/custom.yara

```diff
@@ -197,6 +199,7 @@ rule code_html_1 {
         $html_end = /(^|\n|\>)[ \t]*<\/html/i
 
     condition:
+        mime != "text/xml" and
```

Comment:
Since we are not sending text/xml anymore, this can probably be removed, like the others.
In the case of code_xml, it will never trigger anymore with this check, because text/xml won't be sent.